cmd/limactl/sudoers.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"github.com/lima-vm/lima/pkg/networks"
+	"github.com/urfave/cli/v2"
+)
+
+var sudoersCommand = &cli.Command{
+	Name:   "sudoers",
+	Usage:  "Generate /etc/sudoers.d/lima file.",
+	ArgsUsage: "[SUDOERSFILE]",
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:  "check",
+			Usage: "check that the sudoers file is up-to-date with ~/.lima/_config/networks.yaml",
+		},
+	},
+	Action: sudoersAction,
+}
+
+func sudoersAction(clicontext *cli.Context) error {
```

Comment:
error out when GOOS != "darwin"

Reply:
> error out when GOOS != "darwin"

Done in 97d3a2b9256ab5503ab8176a1741deea48cfb6f8